static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -316,6 +333,12 @@ class EventsTable extends Component<Props, State> {
       );
     const columnOrder = eventView
       .getColumns()
+      .filter(col => {
+        if (!this.state.attachments.length) {
+          return col.key !== 'attachments';
+        }
+        return true;
+      })
```

Comment:
Seems okay, but it's a weird bit of jank that an extra column pops up sometimes and resizes the table. Worth asking design!